elephant/current_source_density.py

```diff
@@ -201,43 +201,52 @@ def estimate_csd(lfp, coords=None, method=None,
-            2D : large_source_2D and small_source_2D
-            3D : gauss_3d_dipole
-        ele_xx : np.array
-            Positions of the x coordinates of the electrodes
-        ele_yy : np.array
-            Positions of the y coordinates of the electrodes
-            Defaults ot None, use in 2D or 3D cases only
-        ele_zz : np.array
-            Positions of the z coordinates of the electrodes
-            Defaults ot None, use in 3D case only
-        x_lims : [start, end]
-            The starting spatial coordinate and the ending for integration
-            Defaults to [0.,1.]
-        y_lims : [start, end]
-            The starting spatial coordinate and the ending for integration
-            Defaults to [0.,1.], use only in 2D and 3D case
-        z_lims : [start, end]
-            The starting spatial coordinate and the ending for integration
-            Defaults to [0.,1.], use only in 3D case
-        res : int
-            The resolution of the integration
-            Defaults to 50
-
-        Returns
-        -------
-        LFP : neo.AnalogSignal object
-           The potentials created by the csd profile at the electrode positions
-           The electrode postions are attached as RecordingChannel's coordinate
+                 x_lims=[0., 1.], y_lims=[0., 1.], z_lims=[0., 1.],
+                 resolution=50):
+    """
+    Forward modelling for getting the potentials for testing Current Source
+    Density (CSD).
+
+    Parameters
+    ----------
+    csd_profile : callable
+        A function that computes true CSD profile.
+        Available options are (see ./csd/utility_functions.py)
+        1D : gauss_1d_dipole
+        2D : large_source_2D and small_source_2D
+        3D : gauss_3d_dipole
+    ele_xx : np.ndarray
+        Positions of the x coordinates of the electrodes
+    ele_yy : np.ndarray, optional
+        Positions of the y coordinates of the electrodes
+        Defaults ot None, use in 2D or 3D cases only
+    ele_zz : np.ndarray, optional
+        Positions of the z coordinates of the electrodes
+        Defaults ot None, use in 3D case only
+    x_lims : list, optional
+        A list of [start, end].
+        The starting spatial coordinate and the ending for integration
+        Defaults to [0.,1.]
+    y_lims : list, optional
+        A list of [start, end].
+        The starting spatial coordinate and the ending for integration
+        Defaults to [0.,1.], use only in 2D and 3D case
+    z_lims : list, optional
+        A list of [start, end].
+        The starting spatial coordinate and the ending for integration
+        Defaults to [0.,1.], use only in 3D case
+    resolution : int, optional
+        The resolution of the integration
+        Defaults to 50
+
+    Returns
+    -------
+    LFP : neo.AnalogSignal
+       The potentials created by the csd profile at the electrode positions.
+       The electrode positions are attached as RecordingChannel's coordinate.
```

Comment:
This sentence is outdated. `RecordingChannels` are removed in current Neo versions in favour of `ChannelIndex` objects.